aws-iot-mitigationaction/aws-iot-mitigationaction.json

```diff
@@ -0,0 +1,239 @@
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-iot.git",
+    "definitions": {
+        "Tag": {
+            "description": "A key-value pair to associate with a resource.",
+            "type": "object",
+            "properties": {
+                "Key": {
+                    "type": "string",
+                    "description": "The tag's key.",
+                    "minLength": 1,
+                    "maxLength": 128
+                },
+                "Value": {
+                    "type": "string",
+                    "description": "The tag's value.",
+                    "minLength": 1,
+                    "maxLength": 256
+                }
+            },
+            "required": [
+                "Value",
+                "Key"
+            ]
+        },
+        "ActionParams": {
+            "type": "object",
+            "properties": {
+                "AddThingsToThingGroupParams": {
+                    "$ref": "#/definitions/AddThingsToThingGroupParams"
+                },
+                "EnableIoTLoggingParams": {
+                    "$ref": "#/definitions/EnableIoTLoggingParams"
+                },
+                "PublishFindingToSnsParams": {
+                    "$ref": "#/definitions/PublishFindingToSnsParams"
+                },
+                "ReplaceDefaultPolicyVersionParams": {
+                    "$ref": "#/definitions/ReplaceDefaultPolicyVersionParams"
+                },
+                "UpdateCACertificateParams": {
+                    "$ref": "#/definitions/UpdateCACertificateParams"
+                },
+                "UpdateDeviceCertificateParams": {
+                    "$ref": "#/definitions/UpdateDeviceCertificateParams"
+                }
+            }
+        },
+        "AddThingsToThingGroupParams": {
+            "description": "Allows adding things that triggered the mitigation action to thing groups",
+            "type": "object",
+            "properties": {
+                "OverrideDynamicGroups": {
+                    "type": "boolean",
+                    "description": "Specifies if this mitigation action can move the things that triggered the mitigation action out of one or more dynamic thing groups."
+                },
+                "ThingGroupNames": {
+                    "description": "The list of groups to which you want to add the things that triggered the mitigation action.",
+                    "type": "array",
+                    "uniqueItems": true,
+                    "items": {
+                        "type": "string",
+                        "minLength": 1,
+                        "maxLength": 128
+                    },
+                    "minItems": 1,
+                    "maxItems":10
+                }
+            },
+            "required": [
+                "ThingGroupNames"
+            ]
+        },
+        "EnableIoTLoggingParams": {
+            "description": "Allows logging",
```

Comment:
Let's use the descriptions from https://docs.aws.amazon.com/iot/latest/apireference/API_MitigationActionParams.html#iot-Type-MitigationActionParams and make sure to have one for every property.

Reply:
Done